components/net/Cargo.toml

```diff
@@ -61,9 +61,8 @@ servo_url = { path = "../url" }
 sha2 = "0.10"
 time = { workspace = true }
 tokio = { version = "1", package = "tokio", features = ["sync", "macros", "rt-multi-thread"] }
```

Comment:
Is package = "tokio" still needed here with the older version removed?

Reply:
likely not... I'm also looking at the wpt failures, so it's not ready yet.